pkg/scalers/prometheus_scaler.go

```diff
@@ -262,6 +273,10 @@ func (s *prometheusScaler) ExecutePromQuery(ctx context.Context) (float64, error
 		url = fmt.Sprintf("%s&namespace=%s", url, s.metadata.namespace)
 	}
 
+	if len(s.metadata.queryParameters) > 0 {
+		url = fmt.Sprintf("%s&%s", url, s.metadata.queryParameters)
+	}
```

Comment:
This requires that the users set the query parameters with the ? and &. I think that this won't work as expected.

Reply:
fixed